internal/providers/terraform/google/registry.go

```diff
@@ -87,6 +88,16 @@ var FreeResources []string = []string{
 	"google_service_account_iam_member",
 	"google_service_account_iam_policy",
 	"google_service_account_key",
+	"google_storage_bucket_access_control",
+	"google_storage_bucket_acl",
+	"google_storage_bucket_iam_policy",
```

Comment:
are we sure this resource is free? [their pricing page](https://cloud.google.com/storage/pricing#operations-pricing) shows `storage.*.setIamPolicy` as a class A operation. Maybe we ignore the resource as free here as it's already included in the bucket cost components so no action is needed. @aliscott any thoughts?

Reply:
I agree with ignoring it here and including it in the bucket for now.